docs/MEV_EDGE_CASES.md

```diff
@@ -0,0 +1,48 @@
+# MEV edge cases
```

Comment:
add a general paragraph about our MEV support, the CLI flag we need to set, a short explanation about how to set the beacon node to use MEV. do we support MEV-boost? do we NOT support certain MEV relays or all are supported? if there are any other known issues I would add them here.

Reply:
Please let me know if the added information is sufficient